File: testprojects/python_main/pipelines/DONOT_OPEN_PYTHON/code/pythonorganization/donot/openme12/graph/Reformat_4.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pythonorganization.donot.openme12.config.ConfigStore import *
from pythonorganization.donot.openme12.udfs.UDFs import *

def Reformat_4(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(lit(10).alias("c_short"), lit(123231).alias("c_int"))

File: testprojects/scala_main/pipelines/DONOT_OPEN_SCALA/code/src/main/scala/org/main/scla_dep_mgmt/graph/all_type_scala_sg_1/recursive_1/Subgraph_1/Reformat_4.scala
package org.main.scla_dep_mgmt.graph.all_type_scala_sg_1.recursive_1.Subgraph_1

import io.prophecy.libs._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.graph.all_type_scala_sg_1.recursive_1.Subgraph_1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_4 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.select(concat(lit(context.config.c_sg1_c_string), lit("hello")).as("c1"))

}
